go.mod

```diff
@@ -1,20 +1,26 @@
 module github.com/GoogleContainerTools/kpt
 
-go 1.17
+go 1.19
+
+replace github.com/GoogleContainerTools/kpt/porch v0.0.7 => ./porch
```

Comment:
IIRC this may break `go get` and `go install` for kpt CLI. Thanks for catching it.
It seems we need to move something from porch to kpt to avoid this replace statement here.
It seems porch's go.mod already has `replace github.com/GoogleContainerTools/kpt => ../`.

Reply:
What is the reason for doing this? If it is to be able to make changes across the different modules more easily, we should look at how we can leverage golang workspaces. 

Also, why would kpt needs to depend on porch? I would assume it should only need to depend on the API? If there are shared components, we should probably either move those to kpt or try to separate it out. But reluctant to add more modules in the repo until we have a better plan for the structure.

And yes, it breaks `go install`.

Reply:
I have moved the oci package a bit now. `replace` is no longer needed.